schemas/groups/propulsion.json

```diff
@@ -64,6 +41,12 @@
       "units": "Hz"
     },
 
+    "maxRevolutions": {
```

Comment:
A good question. I was thinking about the simplest case, `instant` policy with no periods specified where you get updates as new data is received.

I think we have overengineered the subcription protocol and need to simplify.

Btw there's a bit of confusion also related to what goes into context and what in the path - in your example case `vessels.foo` should be in the context, not the path.